examples/api/lib/widgets/value_listenable_builder/value_listenable_builder.0.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [ValueListenableBuilder].
+
+void main() => runApp(const ValueListenableBuilderExampleApp());
+
+class ValueListenableBuilderExampleApp extends StatelessWidget {
+  const ValueListenableBuilderExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: ValueListenableBuilderExample(),
+    );
+  }
+}
+
+class ValueListenableBuilderExample extends StatefulWidget {
+  const ValueListenableBuilderExample({super.key});
+
+  @override
+  State<ValueListenableBuilderExample> createState() => _ValueListenableBuilderExampleState();
+}
+
+class _ValueListenableBuilderExampleState extends State<ValueListenableBuilderExample> {
+  final ValueNotifier<int> _counter = ValueNotifier<int>(0);
+  final Widget goodJob = const Text('Good job!');
```

Comment:
nit: this could be private as well.

Reply:
Actually, this seems unused; so should probably just be removed?

Reply:
Yeah, left over from the code I started from.  Removed.